demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -103,22 +103,26 @@ def is_builtin_condition_task_branches_handled(self, task: Dict) -> bool:
             bool. if the task handles all condition branches correctly.
         """
         is_all_condition_branches_handled: bool = True
+        # ADD all possible conditions to task_condition_labels (UPPER)
         # #default# condition should always exist in a builtin condition
-        task_condition_labels = {'#default#'}
+        task_condition_labels = {'#DEFAULT#'}
         for condition in task.get('conditions', []):
             label = condition.get('label')
             if label:
-                task_condition_labels.add(label)
+                task_condition_labels.add(label.upper())
+
+        # REMOVE all used condition branches from task_condition_labels (UPPWE)
         next_tasks: Dict = task.get('nexttasks', {})
-        for next_task_branch, next_task_ids in next_tasks.items():
+        for next_task_branch in next_tasks.keys():
             try:
-                if next_task_ids:
-                    task_condition_labels.remove(next_task_branch)
+                if next_task_branch:
```

Comment:
why did we change the condition here?

Reply:
While going over the code I noticed this bug. it makes no sense to check the `next_task_ids` since they should always have a value, and we're actually removing the `next_task_branch`